packages/editor/src/components/post-card-panel/index.js

```diff
@@ -106,3 +125,46 @@ export default function PostCardPanel( { className, actions } ) {
 		</PanelBody>
 	);
 }
+
+// Taken from packages/editor/src/components/time-to-read/index.js.
+const AVERAGE_READING_RATE = 189;
+
+// This component renders the wordcount and reading time for the post.
+function PostContentInfo( { postContent } ) {
+	/*
+	 * translators: If your word count is based on single characters (e.g. East Asian characters),
+	 * enter 'characters_excluding_spaces' or 'characters_including_spaces'. Otherwise, enter 'words'.
+	 * Do not translate into your own language.
+	 */
+	const wordCountType = _x( 'words', 'Word count type. Do not translate!' );
+	const wordsCounted = postContent
+		? wordCount( postContent, wordCountType )
```

Comment:
word could is probably a slow function, we should probably avoid recomputing on each render.

Reply:
How can we avoid that though..? Maybe update with a delay?

Reply:
useMemo?

Reply:
You still can't avoid the recalculation though when the content changes.. I thought that what you were talking about..